docs/JestObjectAPI.md

```diff
@@ -921,6 +921,18 @@ When this API is called, all pending macro-tasks and micro-tasks will be execute
 
 This is often useful for synchronously executing setTimeouts during a test in order to synchronously assert about some behavior that would only happen after the `setTimeout()` or `setInterval()` callbacks executed. See the [Timer mocks](TimerMocks.md) doc for more information.
 
+### `jest.runAllTimersAsync()`
+
+Runs all pending timers until there are none remaining.
```

Comment:
Perhaps docs should simply state the async methods are equivalent to the sync ones?

```suggestion
Asynchronous equivalent of `jest.runAllTimers()`.
```